Makefile-Standalone

```diff
@@ -111,8 +111,8 @@ ifeq ($(NO_OPENSSL),1)
 OPENSSL                         = no
 else
 ifeq ($(HOSTOS),darwin)
-OSX_OPENSSL_DIR                 = /usr/local/opt/openssl
-OPENSSL                        ?= $(shell if which brew > /dev/null && brew ls --versions openssl > /dev/null; then echo $(OSX_OPENSSL_DIR); else echo "internal"; fi) 
+OSX_OPENSSL_DIR                 = $(shell if which brew > /dev/null; then brew --prefix; else echo /usr/local; fi)/opt/openssl
+OPENSSL                        ?= $(shell if which brew > /dev/null && brew ls --versions openssl > /dev/null; then echo $(OSX_OPENSSL_DIR); else echo "internal"; fi)
```

Comment:
Rather than changing OSX_OPENSSL_DIR, I suggest eliminating OSX_OPENSSL_DIR altogether and replacing the "then" clause in the OPENSSL shell expression with \`brew --prefix openssl\`
E.g.:

```
if which brew > /dev/null && brew ls --versions openssl > /dev/null; then `brew --prefix openssl`; else echo "internal"; fi)
```
This way brew tells us the full install path, but only when its known to be present on the system.

Reply:
great!